docs/generated/sql/bnf/create_table_as_stmt.bnf

```diff
@@ -1,5 +1,5 @@
 create_table_as_stmt ::=
```

Comment:
can you modify docgen's `diagrams.go` to remove the clause. we're not intending to document it I think.

Reply:
i think the SKIP DOC should do it.